h2o-docs/src/product/data-science/glm.rst

```diff
@@ -103,6 +103,8 @@ Defining a GLM Model
 
 -  `missing_values_handling <algo-params/missing_values_handling.html>`__: Specify how to handle missing values (Skip or MeanImputation).
 
+-  `plug_values <algo-params/plug_values.html>`__: When ``missing_values_handling=plug_values``, specify a single row frame containing values that will be used to impute missing values of the training/validation frame.
```

Comment:
Remove the link to plug_values because we're not adding this to the parameters appendix. The docs will produce an error during the build.